### ISSUE TITLE
Column breaks TextAlign.left

### ISSUE THREAD

**Issue by jimmyff** [DavBfr/dart_pdf#143]
```dart
import 'dart:io';

import 'package:pdf/pdf.dart';
import 'package:pdf/widgets.dart';

void main() {
  final Document pdf = Document();
  Document.debug = true;

  pdf.addPage(Page(build: (context) {
    return Column(
      mainAxisSize: MainAxisSize.min,
      children: [
        Container(
            decoration:
                BoxDecoration(border: BoxBorder(top: true, bottom: true)),
            child: Text('left', textAlign: TextAlign.left)),
        Container(
            decoration:
                BoxDecoration(border: BoxBorder(top: true, bottom: true)),
            child: Text('center', textAlign: TextAlign.center)),
        Container(
            decoration:
                BoxDecoration(border: BoxBorder(top: true, bottom: true)),
            child: Text('right', textAlign: TextAlign.right)),
      ],
    );
  }));

  final File file = File('../test2.pdf');
  file.writeAsBytesSync(pdf.save());
}

```
The above code results in:

<img width="458" alt="Screenshot 2019-09-09 at 19 30 21" src="https://user-images.githubusercontent.com/298850/64556905-4aea2900-d338-11e9-8482-b48956599300.png">

TextAlign.left is not honoured or the container is shrunk so it doesn't occupy the full width. I added borders to demonstrate this.

**Comment by DavBfr** [comment]
use `width: double.infinity,` within Container to use the full width.

**Comment by DavBfr** [comment]
Fixed in Pdf version 1.3.21. Don't hesitate to [Buy Me A Coffee](https://www.buymeacoffee.com/JORBmbw9h).
